eq/deflect/proxy.cpp

```diff
@@ -110,15 +99,29 @@ class Proxy::Impl : public boost::noncopyable
                                               ::deflect::BGRA, offsX, offsY );
         imageWrapper.compressionPolicy = ::deflect::COMPRESSION_ON;
         imageWrapper.compressionQuality = 100;
+        switch( channel.getEye( ))
+        {
+        case eq::EYE_LEFT:
+            imageWrapper.view = ::deflect::View::left_eye;
+            break;
+
+        case eq::EYE_RIGHT:
+            imageWrapper.view = ::deflect::View::right_eye;
+            break;
 
-        _sendFuture = _stream->asyncSend( imageWrapper );
+        default: break;
+        }
+
+        _pending[_current].emplace_back( _stream->asyncSend( imageWrapper ));
```

Comment:
which must be the cause of your stream closing, because then the second finishFrame happens with no data